src/librawspeed/decoders/IiqDecoder.cpp

```diff
@@ -168,6 +180,10 @@ RawImage IiqDecoder::decodeRawInternal() {
   if (width == 0 || height == 0 || width > 11608 || height > 8708)
     ThrowRDE("Unexpected image dimensions found: (%u; %u)", width, height);
 
+  if (split_col < 0 || split_col > width || split_row < 0 || split_row > height)
```

Comment:
`split_col`/`split_row` are unsigned, you can't compare them `<0`.

Reply:
Oh, duh